### PR TITLE
added sftptogo to protocols/index

### PR DESCRIPTION
**Pull request by saggineumann**
Somehow missed this in yesterday's PR 🤦

### CODE REVIEW

**Comment by AliveDevil** [comment]
We could just pay more attention to the build logs 🙃

https://github.com/iterate-ch/docs/pull/346/checks#step:4:236

File: protocols/index.md [hunk 1]
```diff
@@ -21,6 +21,7 @@ onedrive
 sharepoint
 box
 files.com
+sftptogo
```
Comment (dkocher) [inline protocols/index.md:24]: In general we will not want to have a single provider in the top level protocol list. Exceptions apply.

File: protocols/index.md [hunk 2]
```diff
@@ -80,6 +81,9 @@ A highly secure, platform-independent enterprise filesharing solution
 ### [Files.com](files.com.md)
 Files.com is Smart Cloud Storage for modern teams
 
+### [SFTP To Go](sftptogo.md)
```
Comment (dkocher) [inline protocols/index.md:84]: In general we will not want to have a single provider in the top level protocol list. Exceptions apply.

File: protocols/index.md [hunk 3]
```diff
@@ -138,6 +142,7 @@ Protocols with limited support for modification dates only allow to set the modi
 | Microsoft OneDrive   | ✅ ||
 | Microsoft Sharepoint | ✅ ||
 | Files.com            | ✅ ||
+| SFTP To Go           | ✅ ||
```
Comment (dkocher) [inline protocols/index.md:145]: No need to list specific provider as SFTP in general applies.

**Review by dkocher — CHANGES_REQUESTED** [review]
We should add a _Providers_ section to the [SFTP](https://docs.cyberduck.io/protocols/sftp/) page instead as in [WebDAV](https://docs.cyberduck.io/protocols/webdav/#providers).

**Comment by saggineumann** [comment]
@dkocher no problem - I followed files.com's as an example. Shall I go ahead and add the providers section to the SFTP page? Should I move anything besides SFTP To Go in there?

**Comment by dkocher** [comment]
> @dkocher also note that SFTP, FTPS and S3 are all protocols that work with SFTP To Go - should I add providers to these pages too?

We may want to restructure this later.

- [ ] #351

**Comment by saggineumann** [comment]
@dkocher care to review again please?

**Comment by dkocher** [comment]
Looks good to me. Would you mind squashing the two commits.

**Comment by saggineumann** [comment]
squashed

**Review by dkocher — APPROVED** [review]

**Pull request merged**